### PR TITLE
feat(telecom-tracking): track update range phone

### PR DESCRIPTION
**Pull request by varun257**
| Question         | Answers |
|---------------|----------|
| Branch?          | `develop`
| Bug fix?         | no
| New feature?     | no
| Breaking change? | no
| Tickets          | Fix #MANAGER-5088
| License          | BSD 3-Clause

## Description

add tracking to order and update phone range

Signed-off-by: varun257 <varun257@gmail.com>

### CODE REVIEW

File: packages/manager/apps/telecom/src/app/telecom/telephony/line/phone/accessories/choice/accessories-choice.controller.js [hunk 1]
```diff
@@ -115,6 +116,15 @@ export default class TelecomTelephonyLinePhoneAccessoriesChoiceCtrl {
 
   validateStep() {
     this.process.currentView = 'shipping';
+    this.atInternet.trackPage({
```
Comment (marie-j) [inline packages/manager/apps/telecom/src/app/telecom/telephony/line/phone/accessories/choice/accessories-choice.controller.js:119]: Is the `trackPage` needed here ? As there is a hook on `ui router` state change , wan't the page be tracked twice ?

File: packages/manager/apps/telecom/src/app/telecom/telephony/line/phone/accessories/resume/accessories-resume.controller.js [hunk 2]
```diff
@@ -21,7 +22,7 @@ export default class TelecomTelephonyLinePhoneAccessoriesResumeCtrl {
     };
 
     this.loading.init = true;
-    this.process = this.TucTelephonyAccessoriesOrderProcess.getOrderProcess();
+    this.processc = this.TucTelephonyAccessoriesOrderProcess.getOrderProcess();
```
Comment (marie-j) [inline packages/manager/apps/telecom/src/app/telecom/telephony/line/phone/accessories/resume/accessories-resume.controller.js:25]: The variable had the wrong name ?

File: packages/manager/apps/telecom/src/app/telecom/telephony/line/phone/accessories/choice/accessories-choice.controller.js (hunk 1 above)
Comment (varun257) [reply packages/manager/apps/telecom/src/app/telecom/telephony/line/phone/accessories/choice/accessories-choice.controller.js:119]: Its a tab view so the page was not getting tracked

File: packages/manager/modules/telecom-universe-components/src/telecom/v4-links/v4-link/telecom-v4-link.html [hunk 3]
```diff
@@ -51,7 +51,30 @@
...
-        data-ng-if="!$ctrl.action.disabled && $ctrl.action.sref"
+        data-ng-if="!$ctrl.action.disabled && $ctrl.action.sref && $ctrl.action.tracking"
+        data-ui-sref="{{ $ctrl.action.sref }}"
+        title="{{ $ctrl.action.tip || $ctrl.action.text }}"
+        data-track-on="click"
+        data-track-name="{{ $ctrl.action.tracking }}"
+        data-track-type="navigation"
+    >
+        <span class="service-button-description">
+            <i
```
Comment (JeremyDec) [inline packages/manager/modules/telecom-universe-components/src/telecom/v4-links/v4-link/telecom-v4-link.html:62]: Better use a `<span>` tag for this image

File: packages/manager/modules/telecom-universe-components/src/telecom/v4-links/v4-link/telecom-v4-link.html [hunk 4]
```diff
@@ -51,7 +51,30 @@
...
+                class="action-icon ovh-font mr-3"
+                aria-hidden="true"
+                data-ng-if="$ctrl.action.picto"
+                data-ng-class="[$ctrl.action.picto]"
+            ></i>
+            {{ $ctrl.action.text }}
```
Comment (JeremyDec) [inline packages/manager/modules/telecom-universe-components/src/telecom/v4-links/v4-link/telecom-v4-link.html:68]:
```suggestion
            <span data-ng-bind="$ctrl.action.text"></span>
```

File: packages/manager/apps/telecom/src/app/telecom/telephony/line/phone/accessories/resume/accessories-resume.controller.js (hunk 2 above)
Comment (JeremyDec) [reply packages/manager/apps/telecom/src/app/telecom/telephony/line/phone/accessories/resume/accessories-resume.controller.js:25]: Hmm @varun257 I see this as resolved, but there's a typo in the name ?

**Comment by varun257** [comment]
@JeremyDec  It was a typo, have pushed again

**Pull request merged**